src/core_plugins/kibana/public/home/components/recently_accessed.less

```diff
@@ -0,0 +1,9 @@
+.longLink {
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+
+.dropwdownLink {
```

Comment:
Because these names are global, can we scope them with a prefix to avoid a naming collision? E.g. `recentlyAccessedLongLink` and `recentlyAccessedDropdownLink`.